tests/test_to_container.py

```diff
@@ -39,32 +39,57 @@ def assert_container_with_primitives(item: Any) -> None:
 
 
 @mark.parametrize(
-    "cfg,ex_false,ex_true",
+    "src,ex_dict,ex_dict_config,key",
     [
         param(
             {"user": User(age=7, name="Bond")},
             {"user": {"name": "Bond", "age": 7}},
-            {"user": User(age=7, name="Bond")},
+            {"user": OmegaConf.structured(User(age=7, name="Bond"))},
+            "user",
+            id="structured-inside-dict",
         ),
         param(
             [1, User(age=7, name="Bond")],
             [1, {"name": "Bond", "age": 7}],
-            [1, User(age=7, name="Bond")],
-        ),
-        param(
-            {"users": [User(age=1, name="a"), User(age=2, name="b")]},
-            {"users": [{"age": 1, "name": "a"}, {"age": 2, "name": "b"}]},
-            {"users": [User(age=1, name="a"), User(age=2, name="b")]},
+            [1, OmegaConf.structured(User(age=7, name="Bond"))],
+            1,
+            id="structured-inside-list",
         ),
     ],
 )
-def test_exclude_structured_configs(cfg: Any, ex_false: Any, ex_true: Any) -> None:
-    cfg = OmegaConf.create(cfg)
-    ret1 = OmegaConf.to_container(cfg, exclude_structured_configs=False)
-    assert ret1 == ex_false
+class TestSCMode:
```

Comment:
I ended up getting rid of one of the `param`s (there were 3 before, now there are only 2). The 3rd one was not as interesting, and it was hard to figure out what `key` to use for the 3rd one.

Reply:
reading this, I have no idea what you are saying :)

Reply:
Oh, sorry.
There were 3 test cases before. Now there are only 2.